.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ jobs:
       fail-fast: false  # if one fails, do not cancel the other
       matrix:
         gambit-version: ['', '--with-gambit=master']
-        shared-mode: ['', '--enable-shared']
+        shared-mode: ['', '--enable-shared=no']
```

Comment:
Not `--disable-shared` like Gambit? That's confusing.

Reply:
it is canonical, but we can acceot both.

Reply:
done.